reports/queries.py

```diff
@@ -0,0 +1,224 @@
+"""
+This script establishes a connection to the database and provides
+SQL query functions for retrieving key data needed to generate and email the PDF Report.
+"""
+
+import psycopg2
+from psycopg2 import extensions
+from os import environ
+import logging
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def config_log() -> None:
+    """
+    Configure logging for the script.
+    """
+    logging.basicConfig(
+        format="%(asctime)s - %(levelname)s - %(message)s",
+        level=logging.INFO,
+    )
+
+
+def get_db_connection() -> extensions.connection:
+    """
+    Connect to the RDS database.
+    """
+    try:
+        logging.info("Connecting to the database.")
+        connection = psycopg2.connect(
+            host=environ["DB_HOST"],
+            port=environ["DB_PORT"],
+            user=environ["DB_USER"],
+            password=environ["DB_PASSWORD"],
+            database=environ["DB_NAME"]
+        )
+        logging.info("Connected successfully.")
+        return connection
+
+    except psycopg2.OperationalError:
+        logging.warning("The database %s doesn't exist", environ["DB_NAME"])
```

Comment:
What is DB_NAME isn't defined? We don't want the error handling throwing an error